db/migrate/20210226142457_remove_awarding_body_from_application_qualifications.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveAwardingBodyFromApplicationQualifications < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :application_qualifications, :awarding_body, :string
```

Comment:
Including this here may result in some undefined method errors during deploy time. Can we split this into a subsequent PR?

Reply:
Good point, I'll do